ckanext/cnra_schema/schemas/dataset.yaml

```diff
@@ -663,59 +716,6 @@ dataset_fields:
 #- field_name: accessConstraints
 #  label: Access Constraints
```

Comment:
`accessConstraints` should either be moved above `useConstraints` or removed from the schema.